packages/material-ui-lab/src/CalendarPicker/CalendarPicker.test.tsx

```diff
@@ -1,32 +1,35 @@
 import * as React from 'react';
 import { expect } from 'chai';
-import { getClasses, createMount, fireEvent, screen, describeConformance } from 'test/utils';
+import { createMount, fireEvent, screen, describeConformanceV5 } from 'test/utils';
 import LocalizationProvider from '@material-ui/lab/LocalizationProvider';
 import AdapterDateFns from '@material-ui/lab/AdapterDateFns';
-import CalendarPicker from '@material-ui/lab/CalendarPicker';
+import CalendarPicker, { calendarPickerClasses as classes } from '@material-ui/lab/CalendarPicker';
 import { adapterToUse, createPickerRender, getAllByMuiTest } from '../internal/pickers/test-utils';
 
 describe('<CalendarPicker />', () => {
   const mount = createMount();
   const render = createPickerRender({ strict: false });
-  let classes: Record<string, string>;
 
   const localizedMount = (node: React.ReactNode) => {
     return mount(<LocalizationProvider dateAdapter={AdapterDateFns}>{node}</LocalizationProvider>);
   };
 
-  before(() => {
-    classes = getClasses(<CalendarPicker date={adapterToUse.date()} onChange={() => {}} />);
-  });
-
-  describeConformance(<CalendarPicker date={adapterToUse.date()} onChange={() => {}} />, () => ({
+  describeConformanceV5(<CalendarPicker date={adapterToUse.date()} onChange={() => {}} />, () => ({
     classes,
     inheritComponent: 'div',
     render,
+    muiName: 'MuiCalendarPicker',
     mount: localizedMount,
     refInstanceof: window.HTMLDivElement,
     // cannot test reactTestRenderer because of required context
-    skip: ['componentProp', 'propsSpread', 'reactTestRenderer'],
+    skip: [
+      'componentProp',
+      'componentsProp',
+      'propsSpread',
+      'reactTestRenderer',
+      'themeDefaultProps',
+      'themeVariants',
```

Comment:
Let's add TODO indicating why it is skipped (not spreading props on root element)

Reply:
added.